### PR TITLE
Adding jsonschema typings

### PR DESCRIPTION
**Pull request by vladotesanovic**
case 1. Add a new type definition.
- [x] checked compilation succeeds with `--target es6` and `--noImplicitAny` options.
- [x] has correct [naming convention](http://definitelytyped.org/guides/contributing.html#naming-the-file)
- [x] has a [test file](http://definitelytyped.org/guides/contributing.html#tests) with the suffix of  `-tests.ts` or `-tests.tsx`.

case 2. Improvement to existing type definition.
- documentation or source code reference which provides context for the suggested changes.  url http://api.jquery.com/html .
  - it has been reviewed by a DefinitelyTyped member.

### CODE REVIEW

File: jsonschema/jsonschema.d.ts [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+         */
+        getSchema(urn: string): {};
+
+        /**
+         * Validates an instance against the schema (the actual work horse)
+         * @param instance
+         * @param schema
+         * @param options
+         * @param ctx
+         * @private
```
Comment (sandersn) [inline jsonschema/jsonschema.d.ts]: If this is private, shouldn't it be removed from Validator's interface?

Comment (vladotesanovic) [inline jsonschema/jsonschema.d.ts]: you are right, that on is useless. I ll update stuff

**Comment by e-cloud** [comment]
@vladotesanovic , this is not JsonSchema itself, it's just JsonSchema's validator. Since this take the place, how could i add the definition for the schema itself?

**Comment by e-cloud** [comment]
@vladotesanovic , this is not JsonSchema itself, it's just JsonSchema's validator. Since this take the place, how could i add the definition for the schema itself?

**Pull request merged**